### PR TITLE
Setup the docs page containing all the cheat-sheets. 

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
image

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+PyAnsys cheat-sheet
```
Comment (RobPasMue) [inline doc/source/index.rst:1]:
```suggestion
PyAnsys cheat-sheets
```

File: doc/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+===================
```
Comment (RobPasMue) [inline doc/source/index.rst:2]:
```suggestion
====================
```

File: doc/source/index.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            
+            :octicon:`file;2em;sd-text-info`
+
+
+    .. grid-item::
+
+        .. card:: pyaedt cheat-sheet
+            :link: https://cheatsheet.mapdldocs.pyansys.com/pyaedt_cheat_sheet.pdf
+            
+            :octicon:`file;2em;sd-text-info`
```
Comment (RobPasMue) [inline doc/source/index.rst:24]:
```suggestion
        .. card:: PyAEDT
            :link: https://cheatsheets.docs.pyansys.com/pyaedt_cheat_sheet.pdf
            
            :octicon:`file;2em;sd-text-info`
```

File: doc/source/index.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+PyAnsys general cheat-sheet template.
+-------------------------------------
+
+.. grid:: 2 1 1 2
+
+    .. grid-item::
+        
+        .. card:: pyansys-cheat-sheat template
+            :link: https://cheatsheet.mapdldocs.pyansys.com/pyansys_cheat_sheet.pdf
```
Comment (RobPasMue) [inline doc/source/index.rst:34]:
```suggestion
        .. card:: Cheat-sheat template
            :link: https://cheatsheets.docs.pyansys.com/pyansys_cheat_sheet.pdf
```

File: doc/source/index.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    .. grid-item::
+
+        .. card:: pyaedt cheat-sheet
+            :link: https://cheatsheet.mapdldocs.pyansys.com/pyaedt_cheat_sheet.pdf
+            
+            :octicon:`file;2em;sd-text-info`
+
+PyAnsys general cheat-sheet template.
+-------------------------------------
```
Comment (RobPasMue) [inline doc/source/index.rst:27]:
```suggestion
PyAnsys general cheat-sheet template
------------------------------------
```

**Review by RobPasMue — APPROVED** [review]
LGTM! Some minor naming comments. Thanks @Revathyvenugopal162

File: .github/workflows/ci_cd.yml [hunk 6]
```diff
@@ -20,30 +20,48 @@ jobs:
...
+        #if: github.ref == 'refs/heads/main'
         run: |
           mkdir build-pdf
-          cp build/*.pdf build-pdf/
+          cp _build/*.pdf build-pdf/
+          cp -r doc/_build/html/* build-pdf/
 
       - name: Deploy to gh-pages
-        if: github.ref == 'refs/heads/main'
+        #if: github.ref == 'refs/heads/main'
```
Comment (Revathyvenugopal162) [inline .github/workflows/ci_cd.yml:64]:
```suggestion
        if: github.ref == 'refs/heads/main'
```

File: .github/workflows/ci_cd.yml [hunk 7]
```diff
@@ -20,30 +20,48 @@ jobs:
...
         with:
-          name: Cheat-Sheets
-          path: build/*.pdf
+          name: documentation-pdf
+          path: _build/*.pdf
           retention-days: 7
 
       - name: Prepare files for gh-pages deployment
-        if: github.ref == 'refs/heads/main'
+        #if: github.ref == 'refs/heads/main'
```
Comment (Revathyvenugopal162) [inline .github/workflows/ci_cd.yml:57]:
```suggestion
        if: github.ref == 'refs/heads/main'
```

**Pull request merged**